smartapps/samsung/sleepsense-wake-up.src/sleepsense-wake-up.groovy

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
**OVERALL COMMENT**
This can be combined with 1, 2, 3 SmartApps. The sleepstate/bedstate to automate lights can all be set up in one smartapp.